
This patch moves webbrowser windowwidget after progress bar widget to reduce a bit
issue of flicking window in mythwebbrowser. 
Patch only redfuces flicking issue - not solves it completelly :-(


diff --speed-large-files --no-dereference --minimal -Naur Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/browser-ui.xml Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/browser-ui.xml
--- Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/browser-ui.xml	2022-04-26 01:09:17.000000000 +0200
+++ Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/browser-ui.xml	2022-03-17 16:01:19.635148210 +0100
@@ -13,15 +13,15 @@
         <progressbar name="progressbar" from="baseprogress_s">
             <area>12, 52,-12,12</area>
         </progressbar>
-        <webbrowser name="webbrowser" from="basebrowser">
-            <area>0,65,100%,-50</area>
-            <browserarea>12,0,100%-12,100%-12</browserarea>
-        </webbrowser>
         <textarea name="status" from="basetext_s">
             <area>50,100%-50,-50,100%</area>
             <align>center</align>
             <alpha>150</alpha>
         </textarea>
+        <webbrowser name="webbrowser" from="basebrowser">
+            <area>0,65,100%,-50</area>
+            <browserarea>12,0,100%-12,100%-12</browserarea>
+        </webbrowser>
     </window>
 
     <window name="bookmarkmanager">
